build.js

```diff
@@ -9,26 +10,36 @@ const stringifyObject = require('stringify-object');
 
 const pkg = require('./package.json');
 
-let array = Object.keys(require('spdx-license-list'));
-let files = [
-  {
-    path: pkg.main,
-    contents: JSON.stringify(array, null, '  ')
-  },
-  {
-    path: require('./bower.json').main,
-    contents: `window.spdxLicenseIds = ${stringifyObject(array, {indent: '  '})};`
-  }
-];
+http.get('http://spdx.org/licenses/licenses.json', response => {
+  let buffer = '';
+  response
+    .on('data', data => {
+      buffer += data;
+    })
+    .on('end', () => {
+      const json = JSON.parse(buffer);
```

Comment:
This code could be a lot simpler if you used [`got`](https://github.com/sindresorhus/got) ;)